packages/terra-status-view/tests/jest/StatusView.test.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import Button from 'terra-button';
+import { IntlProvider } from 'react-intl';
+import StatusView from '../../src/StatusView';
+import messages from '../../translations/en-US.json';
+
+const locale = 'en-US';
+
+// Snapshot Tests
+it('should render an image with a default status view', () => {
+  const defaultRender = (<IntlProvider locale={locale} messages={messages}>
```

Comment:
Check out the intl-context-setup file that @emilyrohrbough wrote in terra-form-field: https://github.com/cerner/terra-core/blob/master/packages/terra-form-field/tests/jest/intl-context-setup.js